cloudify_agent/shell/commands/daemons.py

```diff
@@ -133,6 +167,38 @@ def create(**params):
     attributes.update(_parse_custom_options(custom_arg))
     click.echo('Creating...')
     from cloudify_agent.shell.main import get_logger
+
+    if attributes['broker_get_settings_from_manager']:
+        # We have to build the client manually as the rest client
+        # expects us to have the env variable set for the cloudify
+        # manager IP, and it is not always set
+        client = CloudifyClient(
+            attributes['manager_ip'],
+            attributes['manager_port'],
+        )
+        # Get the broker user and password from the manager
+        bootstrap_context_dict = client.manager.get_context()
+        bootstrap_context_dict = bootstrap_context_dict['context']['cloudify']
+        bootstrap_context = BootstrapContext(bootstrap_context_dict)
+        bootstrap_agent = bootstrap_context.cloudify_agent
+
+        broker_user, broker_pass = utils.internal.get_broker_credentials(
+            bootstrap_agent
+        )
+
+        try:
+            attributes['broker_user'] = broker_user
+            attributes['broker_pass'] = broker_pass
+            attributes['broker_ssl_enabled'] = \
+                bootstrap_agent.broker_ssl_enabled
+            attributes['broker_ssl_cert'] = bootstrap_agent.broker_ssl_cert
+        except AttributeError:
```

Comment:
It might help in the case of some local testing where the bootstrap agent isn't really a bootstrap agent but instead, e.g. an empty dict.

If that's not a concern here then it should be safe to remove.